Microsoft.Toolkit.Uwp.UI/Shadows/AttachedDropShadow.cs

```diff
@@ -166,6 +167,16 @@ protected internal override void OnElementContextUninitialized(AttachedShadowEle
                 _container.Children.Remove(context.SpriteVisual);
             }
 
+            context.SpriteVisual?.StopAnimation("Size");
```

Comment:
@Sergio0694 any other clean-up I need to do beyond this to clean-up the composition animation?

Reply:
Don't think so, the actual objects would just be disposed normally, we're not doing any specific cleanup in other places of the animation package either, it's mostly just fire and forget as usual. Stopping these animations is the only important bit because otherwise the composition layer will just keep them always active even if the XAML object is gone, but other than that we should be good 😄